transaction/atomic.go

```diff
@@ -148,6 +169,10 @@ func AtomicTransfer(hezClient client.HermezClient, ethereumChainID int,
 
 	// Post
 	serverResponse, err = ExecuteAtomicTransaction(hezClient, atomicGroup)
```

Comment:
Name suggestion:

```suggestion
	serverResponse, err = SendAtomicGroup(hezClient, atomicGroup)
```

keep in mind that in the future we may allow to send single atomic transactions (instead of groups of txs), and the current name seems to talk about a single tx